src/main/java/com/trillia/entities/Contact.java

```diff
@@ -0,0 +1,107 @@
+package com.trillia.entities;
+
+import org.hibernate.search.annotations.*;
+import org.hibernate.search.annotations.Index;
+
+import javax.persistence.*;
+import javax.xml.bind.annotation.XmlElement;
+import java.io.Serializable;
+
+/**
+ * Created by mmathew on 27/05/16.
+ */
+
+
+@Entity
+@Table(name = "trl_contacts", uniqueConstraints = {@UniqueConstraint(columnNames = {"tenantId"}), @UniqueConstraint(columnNames = {"contactAlias"})})
+@Indexed(index = "Contacts")
```

Comment:
index name need not be matching table names. Infact we can have few index groups e.g. MasterDataIndex , SetupDataIndex, PoDataIndex, WarehouseDataIndex , etc. module wise. The current one may belong to SetupDataIndex.